packages/IMAPClient-UI.package/ICFolderDialog.class/instance/modifyFlaggedFlag.st

```diff
@@ -1,7 +0,0 @@
-flags
-modifyFlaggedFlag
-
-	selectedEmail modifyFlag: #flagged.
-	selectedEmail flagged: (selectedEmail retrieveFlags at: #flagged).
-	selectedEmail saveInMaildir.
-	self changed: #emailsOfSelectedFolder
```

Comment:
small refactoring that is not really part of this issue, but it is nice to have:)